tutorials/lodash/remove-null-from-array.md

```diff
@@ -0,0 +1,65 @@
+To remove a null from an array, you should use lodash's `filter` function.
+It takes two arguments:
+
+- Collection, the object or array to iterate over.
+- Predicate, the function invoked per iteration.
+
+It returns the new filtered array.
```

Comment:
This sentence conveys no useful info and doesn't tie back to the title of the tutorial.

"The `find()` function returns a new array containing all elements `predicate` returned a truthy value for. To remove `null`, you can call `find()` with `v => v !== null` as the `predicate`."